ethereum/contracts/ETHApp.sol

```diff
@@ -47,6 +50,8 @@ contract ETHApp is RewardSource, AccessControl {
         c2.inbound = _incentivized.inbound;
         c2.outbound = _incentivized.outbound;
 
+        _setupRole(CHANNEL_UPGRADE_ROLE, msg.sender);
+        _setRoleAdmin(INBOUND_CHANNEL_ROLE, CHANNEL_UPGRADE_ROLE);
```

Comment:
still need to look into openzep code properly, but it seems like _setRoleAdmin basically makes it that only those with CHANNEL_UPGRADE_ROLE can change the INBOUND_CHANNEL_ROLE in future?

without this, does it default to letting anyone change it, or letting noone change it?

Reply:
defaults to "let no one change it"